mobSpawnerTests.js

```diff
@@ -0,0 +1,92 @@
+/*
+ * tests for the spawner object
+ */
+
+import { Spawner } from './src/classes/spawner';
+class spawnerTester{
+    constructor(){
+
+    }
+
+    spawnerTest(){
+        var results = []

```

Comment:
Stay away from var, use let instead 
```suggestion
let results = []
```